graylog2-web-interface/src/pages/ErrorPage.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, Jumbotron, Row } from 'react-bootstrap';
+import { DocumentTitle } from 'components/common';
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import style from '!style/useable!css!./NotFoundPage.css';
+import errorPageStyles from './ErrorPage.css';
+
+class ErrorPage extends React.Component {
+  static propTypes = {
+    error: PropTypes.shape({
+      message: PropTypes.string.isRequired,
+    }).isRequired,
+    info: PropTypes.shape({
+      componentStack: PropTypes.string.isRequired,
+    }).isRequired,
+  };
+
+  componentDidMount() {
+    style.use();
+  }
+
+  componentWillUnmount() {
+    style.unuse();
+  }
+
+  render() {
+    const { error, info } = this.props;
+
+    return (
+      <DocumentTitle title="Not Found">
+        <Row className="jumbotron-container">
+          <Col mdOffset={2} md={8}>
+            <Jumbotron>
+              <h1>Something went wrong.</h1>
+              <p>It seems like the page you navigated to contained an error.</p>
+              <p>You can use the navigation to reach other parts of the product, refresh the page or submit an error report.</p>
```

Comment:
Just a suggestion: Would here not be a good place for the issue page on github?
It could be a bit crowded, but than we could also ask to the user to add the stacktrace to the issue.

Reply:
I was also thinking about this, but I was hesitating because the error could also be the result of a 3rd party plugin that is faulty.

Reply:
That is true, but don't we not still know if one of our used libraries make graylog unusable?

Reply:
Sure. But we cannot differentiate where it happened and we might produce a lot of issues in the wrong repository.

Nevertheless, my suggestion is to include the same component that we show when a search term returns no results/fails:

![screen shot 2018-09-26 at 11 45 42](https://user-images.githubusercontent.com/41929/46072128-db5c4780-c181-11e8-9959-431e67b14212.png)

This way we offer the user some options to proceed with the error and not just "open an issue here".

Reply:
I think adding such a component would make sense. I would also display more clearly the Graylog version being used, and by default collapse the stack trace and component tree, as they may be too prominent and scary, specially if users are not used to debugging computer applications.

Additionally I think we should also consider making it easier to submit this information to us by adding a "copy to clipboard" button to get all the information (maybe even formatted as markdown).